doc/source/management/data_at_rest_encryption.rst

```diff
@@ -291,72 +294,58 @@ The implementation of the behavior controlled by this variable is considered
 This variable was ported from MariaDB and then extended to support key rotation. This
 variable has the following possible values:
 
-.. rubric:: ON
-
-New tables are created encrypted. You can create an unencrypted table by using
-the ``ENCRYPTION=NO`` clause to the ``CREATE TABLE`` or ``ALTER TABLE``
-statement.
-
-.. rubric:: OFF
-
-By default, newly created tables are not encrypted. Add the ``ENCRYPTION=NO``
-clause in the ``CREATE TABLE`` or ``ALTER TABLE`` statement to create an
-encrypted table.
-
-.. rubric:: FORCE
-
-New tables are created encrypted with the master key. Passing ``ENCRYPTION=NO``
-to ``CREATE TABLE`` or ``ALTER TABLE`` will result in an error and the table
-will not be created or altered.
-
-If you alter a table which was created without encryption, note that it will not
-be encrypted unless you use the ``ENCRYPTION`` clause explicitly.
-
-.. rubric:: KEYRING_ON
-
-:Availability: This value is **Alpha** quality
-
-New tables are created encrypted with the keyring as the default encryption. You
-may specify a numeric key identifier and use a specific ``percona-innodb-`` key
-from the keyring instead of the default key:
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Value
+     - Description
+   * - ON
+     - New tables are created encrypted. You can create an unencrypted table by using
+       the ``ENCRYPTION=NO`` clause to the ``CREATE TABLE`` or ``ALTER TABLE``
+       statement.
+   * - OFF
+     - By default, newly created tables are not encrypted. Add the ``ENCRYPTION=NO``
+       clause in the ``CREATE TABLE`` or ``ALTER TABLE`` statement to create an
+       encrypted table.
+   * - FORCE
+     - New tables are created encrypted with the master key. Passing
+       ``ENCRYPTION=NO`` to ``CREATE TABLE`` or ``ALTER TABLE`` will result in
+       an error and the table will not be created or altered. If you alter a
+       table which was created without encryption, note that it will not be
+       encrypted unless you use the ``ENCRYPTION`` clause explicitly.
+   * - KEYRING_ON (This value is **Alpha** quality)
+     - New tables are created encrypted with the keyring as the default encryption. 
+   * - FORCE_KEYRING (This value is **Alpha** quality)
+     - New tables are created encrypted and keyring encryption is enforced.
+   * - ONLINE_TO_KEYRING (This value is **Alpha** quality)
+     - All tables created or altered without the ``ENCRYPTION=NO`` clause are
+       encrypted with the latest version of the default encryption key. If a table
```

Comment:
It should say that they are encrypted with keyring encryption. Also there are two cases for encryption key. A table can have assigned encryption_key_id without being encrypted. For instance CREATE TABLE t1 ENCRYPTION_KEY_ID = 6, or with alter : ALTER TABLE t1 ENCRYPTION_KEY_ID = 3. If a table has an encryption key id assigned then this key will be used by encryption threads to encrypt the table. Otherwise the GLOBAL value of innodb_default_encryption_key_id will be used as a key for encryption.